CHANGES.md

```diff
@@ -126,6 +126,11 @@ user-facing, though:
 * Prover tactics (e.g., `yices`) now return `ProofScript ()` instead of
   `ProofScript SatResult`.
 
+* `Simpset`s can now contain "permutative" rewrite rules, where a rule
+  is permutative if each side of the equality is an instance of the
```

Comment:
I don't think I know what "instance" means here

Reply:
Maybe saying "if each side of the equality could match the same set of terms and therefore the rule could repeatedly apply forever" would be better?